src/user.rs

```diff
@@ -99,22 +130,27 @@ pub trait UserOps {
     fn user_get_public_key(&self, users: &[UserId]) -> Result<HashMap<UserId, PublicKey>>;
 }
 impl UserOps for IronOxide {
-    fn user_verify(jwt: &str) -> Result<Option<UserVerifyResult>> {
-        let mut rt = Runtime::new().unwrap();
-        rt.block_on(user_api::user_verify(jwt.try_into()?, OUR_REQUEST))
-    }
-
-    fn user_create(jwt: &str, password: &str) -> Result<UserCreateKeyPair> {
+    fn user_create(
+        jwt: &str,
+        password: &str,
+        user_create_opts: &UserCreateOpts,
+    ) -> Result<UserCreateKeyPair> {
         let recrypt = Recrypt::new();
         let mut rt = Runtime::new().unwrap();
         rt.block_on(user_api::user_create(
             &recrypt,
             jwt.try_into()?,
             password.try_into()?,
+            user_create_opts.needs_rotation,
             OUR_REQUEST,
         ))
     }
 
+    fn user_list_devices(&self) -> Result<UserDeviceListResult> {
+        let mut rt = Runtime::new().unwrap();
+        rt.block_on(user_api::device_list(self.device.auth()))
+    }
```

Comment:
Did we just forget to expose this function or something?

Reply:
No, it just got moved.

Reply:
Sorry for the confusion. We were getting a linting warning about our method order not matching the trait order. I fixed it in this commit: https://github.com/IronCoreLabs/ironoxide/pull/43/commits/c68eb406b6054a2322c1d456339b5fe4602f5aec

Churn!